module-api/src/main/java/ru/feathercore/moduleapi/ModuleLoader.java

```diff
@@ -62,4 +62,15 @@
      * @apiNote modifications to returned collection should not have side-effect on this module loader
      */
     Collection<? extends M> getModules();
+
+    /**
+     * Checks whether module of given class was loaded by this module loader.
+     *
+     * @param module class of module to be checked.
+     * @return true/false.
```

Comment:
Provide more exact description for `@return` also using `{@link true}` and `{@link false}`